phoenix-core-client/src/main/java/org/apache/phoenix/jdbc/PhoenixStatement.java

```diff
@@ -825,14 +826,14 @@ private ExecutableSelectStatement(TableNode from, HintNode hint, boolean isDisti
                 boolean hasSequence, List<SelectStatement> selects, Map<String, UDFParseNode> udfParseNodes) {
             super(from, hint, isDistinct, select, where, groupBy, having, orderBy, limit, offset, bindCount, isAggregate, hasSequence, selects, udfParseNodes);
         }
-        
+
         private ExecutableSelectStatement(ExecutableSelectStatement select) {
             this(select.getFrom(), select.getHint(), select.isDistinct(), select.getSelect(), select.getWhere(),
                     select.getGroupBy(), select.getHaving(), select.getOrderBy(), select.getLimit(), select.getOffset(), select.getBindCount(),
                     select.isAggregate(), select.hasSequence(), select.getSelects(), select.getUdfParseNodes());
         }
-		
```

Comment:
Too many formatting changes, can we avoid it? :)

Reply:
I would like to keep them as they will fix a lot of Yetus warnings. One general argument is that such changes should be done in a dedicated commit, but that never happens. You can however ignore whitespace changes:

<img width="223" height="190" alt="image" src="https://github.com/user-attachments/assets/632bf511-8e07-4be8-9aa7-c4d4bdb3351d" />

Here is the link for your convenience: https://github.com/apache/phoenix/pull/2226/files?diff=unified&w=1

Honestly, I don't know how to revert just the whitespace changes, they were done by the editor settings to strip trailing whitespace while saving. Let me know if you still feel strongly about it, I will try to figure out how to do it.